### DISCUSSION TITLE
About the category

### DISCUSSION THREAD

**Discussion by Kixunil** [rust-bitcoin/rust-bitcoin#816] [Other Bitcoin-related crates]
External Open Source libraries may announce themselves here to get a bit more visibility.
However the maintainers of rust-bitcoin **do NOT guarantee the crates here are non-malicious, secure or bug-free!**
Obvious offenders will be removed when possible.

The main idea is that several people created libraries external to rust-bitcoin that, for various reasons, are not suitable for inclusion in `rust-bitcoin` but could still be useful for some developers and a place to tell about them should make it easier for others to learn about them.

**Comment by dr-orlovsky** [comment]
I manage many repos which depende on `rust-bitcoin`. Will name just the most end-user-facing:

- [Descriptor wallet](https://github.com/LNP-BP/descriptor-wallet): library and command-line wallet strictly separating private and public keys and enabling arbitrary complex descriptors
- [Bitcoin Pro](https://github.com/pandoracore/bitcoin-pro): GTK-based UI tool for read-only descriptor-based wallets and blockchain exploring
- [LNP Node](https://github.com/LNP-BP/lnp-node): Lightning node in rust written from scratch independently from `rust-lightning`/LDK
- [BP Core](https://github.com/LNP-BP/bp-core): Bitcoin protocol core library implementing primitives for client-side-validation like Peter Todd's single-use-seals and deterministic bitcoin commitments based on P2C, S2C and other tweaks
- [RGB Core](https://github.com/rgb-org/rgb-core): library implementing RGB (smart contracts using client-side-validation for bitcoin and lightning network)
